addons/sys_rack/stringtable.xml

```diff
@@ -183,5 +183,11 @@
             <Spanish>Dejar de usar todas</Spanish>
             <French>Arrêter de tout utiliser</French>
         </Key>
+        <Key ID="STR_ACRE_sys_rack_turnOnSpeaker">
+            <English>Turn On Speaker</English>
```

Comment:
Does maybe `Speaker On` and `Speaker Off` fit better with the rest of action naming? (I am not sure.)